server/routes/notification.js

```diff
@@ -6,10 +6,12 @@ const {
   createNotification,
   getAllNotifications,
   deleteNotification,
+  updateNotification
 } = require("../controllers/notification");
 
 router.route("/").post(protect, createNotification);
 router.route("/").get(protect, getAllNotifications);
 router.route("/:id").delete(protect, deleteNotification);
+router.route("/update/:id").post(protect, updateNotification);
```

Comment:
Please see comment above about HTTP methods

Reply:
made changes in the recent commit